lib/puppet/functions/redis/get.rb

```diff
@@ -0,0 +1,32 @@
+require 'redis'
+# Returns the value of the key being looked up or `undef` if the key does not
+# exist. Takes two arguments with an optional third. The first being a string
+# value of the key to be looked up, the second is the URL to the Redis service
+# and the third optional argument is a default value to be used if the lookup
+# fails.
+#
+# example usage
```

Comment:
Unfortunately, https://github.com/puppetlabs/puppet-strings/pull/188 still isn't in a released version of puppet-strings.

Reply:
@eputnam Hi! I know you don't work on it anymore, but is there anyone you can suggest pinging to get a new puppet-strings release out?